### PR TITLE
Add Custom Favicon

### PR DESCRIPTION
**Pull request by reynaldiarya**
added a custom favicon to the settings system

### CODE REVIEW

File: src/install/sql/content.sql [hunk 1]
```diff
@@ -322,28 +322,29 @@ VALUES
...
-	(20,'invoice_refund_logic','credit_note',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(21,'invoice_cn_series','CN-',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(22,'invoice_cn_starting_number','1',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(23,'nameserver_1',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(24,'nameserver_2',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(25,'nameserver_3',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(26,'nameserver_4',NULL,0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(27,'funds_min_amount','10',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00'),
-	(28,'funds_max_amount','200',0,NULL,NULL,'2022-12-01 12:00:00','2022-12-01 12:00:00');
+	(7,'company_favicon','themes/huraga/assets/favicon.ico',0,NULL,NULL,'2023-01-08 12:00:00','2023-01-08 12:00:00'),
```
Comment (jaapmarcus) [inline src/install/sql/content.sql:325]: Why not append it as 29?

Also missing upgrade script

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

File: src/install/sql/content.sql (hunk 1 above)
Comment (reynaldiarya) [reply src/install/sql/content.sql:325]: I think it's better close to the logo if it's opened via the mysql management ui

where do i find the upgrade script

Comment (jaapmarcus) [reply src/install/sql/content.sql:325]: > I think it's better close to the logo if it's opened via the mysql management ui
> 
> where do i find the upgrade script

https://github.com/FOSSBilling/FOSSBilling/blob/3c793a630eacd7ce3713c49bd9c7b8cb7c5fb6eb/src/foss-update.php#L27

The location in the form is hardcoded set and not based on the order... Aand for users who are willing to change it via phpmyadmin it should not matter  they are used by the pain full way

File: src/foss-update.php [hunk 2]
```diff
@@ -35,6 +35,16 @@ public function patch()
     }
 }
 
+class FOSSPatch_Favicon extends FOSSPatchAbstract
```
Comment (jaapmarcus) [inline src/foss-update.php:38]: Naming of the class should be FOSSpatch_26 in this case 
See
https://github.com/boxbilling/boxbilling/blob/master/src/bb-update.php

Also it should be before FOSSPatch_25

**Review by jaapmarcus — APPROVED** [review]

**Review by BelleNottelling — APPROVED** [review]

**Pull request merged**